src/main/java/github/weichware10/util/data/TrialData.java

```diff
@@ -22,44 +34,67 @@
     private List<DataPoint> dataPoints;
 
     /**
-     * Stores the TrialData for the different tools internally.
-     *
-     * @param toolType - the tooltype of the stored data
-     * @param trialId  - the id of the trial
-     * @param configId - the configuration of the stored data
+     * Konstruktor für Jackson.
      *
-     * @since v0.2
+     * @param toolType  - the tooltype of the stored data
+     * @param trialId   - the id of the trial
+     * @param configId  - the configuration of the stored data
+     * @param startTime - Startzeitpunkt des Versuchs
      */
-    public TrialData(Enums.ToolType toolType, String trialId, String configId) {
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public TrialData(@JsonProperty("toolType") Enums.ToolType toolType,
+            @JsonProperty("trialId") String trialId, @JsonProperty("configId") String configId,
+            @JsonProperty("startTime") DateTime startTime, @JsonProperty("answer") String answer,
+            @JsonProperty("data") List<DataPoint> dataPoints) {
         this.toolType = toolType;
         this.trialId = trialId;
         this.configId = configId;
-        this.startTime = DateTime.now();
-        this.dataPoints = new ArrayList<DataPoint>();
+        this.startTime = startTime;
+        this.answer = answer;
+        this.dataPoints = dataPoints;
     }
 
     /**
      * Stores the TrialData for the different tools internally.
      *
-     * @param toolType   - the tooltype of the stored data
-     * @param trialId    - the id of the trial
-     * @param configId   - the configuration of the stored data
-     * @param startTime  - Startzeitpunkt des Versuchs
-     * @param answer     - Anwort des Versuchs
-     * @param dataPoints - Daten des Versuchs
+     * @param toolType - the tooltype of the stored data
+     * @param trialId  - the id of the trial
+     * @param configId - the configuration of the stored data
      *
-     * @since v0.3
+     * @since v0.2
```

Comment:
Warum gehen wir in der Versionen zurück ?

Reply:
Ist ein Bug da Github irgendwie zwei verschiedene Sachen vergleicht, da ich den anderen Konstruktor als Kommentar gesetzt habe der vorher da stand. v0.2 ist schon richtig

Reply:
achso, mal schauen ob sich das klärt wenn du die andern löschst

Reply:
nope bei mir ist das immer noch so